Marlin/SanityCheck.h

```diff
@@ -292,7 +292,15 @@
   #endif
 
   /**
+   * Check if ENDSTOPS_ONLY_FOR_HOMING is disabled when M211 command is enabled
+   */
+  #if (defined(ENABLE_M211)) && (!defined(ENDSTOPS_ONLY_FOR_HOMING))
+    #error ENDSTOPS_ONLY_FOR_HOMING must be enabled when M211 command is active.
```

Comment:
Good to have, but since `ENDSTOPS_ONLY_FOR_HOMING` is enabled in `Conditionals.h` when `ENABLE_M211` is enabled, this error will never be thrown.